app/translations/en.json

```diff
@@ -14,6 +14,7 @@
   "donation": "Thanks for supporting Zingo!",
   "restarting": "Error connecting to the Server, the App will restart in a few seconds.",
   "change-privacy": "New privacy policy: ",
+  "change-privacy-legend": ". Tap in any amount as: 'ZEC -.- - - -' or '$ -.- -' to reveal the value.",
```

Comment:
i think it is easier to read if it just says "Tap to reveal hidden values"
i find the amount of words and the examples harder to understand personally